internal/npm_install/npm_install.bzl

```diff
@@ -54,6 +54,14 @@ filegroup(
           repository_ctx.attr.package_lock_json,
           repository_ctx.path("package-lock.json"))
 
+  # Add data dependencies to the repository
+  for f in repository_ctx.attr.data:
+    to = []
+    if f.package:
+      to += [f.package]
+    to += [f.name]
+    repository_ctx.symlink(f, repository_ctx.path("/".join(to)))
```

Comment:
does it work on Windows? seems like this relies on file symlinks?

Reply:
It does. Not sure what bazel does but maybe it just copies the file instead of symlinking it. Same function is used to get 'package.json' into the repository.

Reply:
```
  repository_ctx.symlink(
      repository_ctx.attr.package_json,
      repository_ctx.path("package.json"))
```

Reply:
okay, interesting